Resources/Prototypes/Entities/Objects/Fun/immovable_rod.yml

```diff
@@ -177,3 +177,15 @@
     state: icon
     rotation: 225
     noRot: false
+
+- type: entity
+  parent: ImmovableRodKeepTilesStill
+  id: ImmovableRodWeh
+  name: immovable weh
+  description: WEH!
+  components:
+  - type: Sprite
+    sprite: Objects/Fun/toys.rsi
+    state: plushie_lizard
+    rotation: 225
```

Comment:
does it have to be rotated? The lizard sprite should already be upright, unlike the throngler or gibstick

Reply:
It doesn't have to be, but I figure it moves fast enough that to see it turned in the first place would be incredibly rare